draft-ietf-rats-architecture.md

```diff
@@ -666,6 +649,49 @@ network Attester.
 In essence, an entity that combines more than one role also creates and
 consumes the corresponding conceptual messages as defined in this document.
 
+# Role Hosting and Composition
+
+NED VERSION
+
+The RATS architecture includes the definition of Roles (e.g. Attester, Verifier,
+Relying Party, Endorser) and conceptual messages (e.g., Evidence, Attestation Results,
+Endorsements, Appraisal Policies) that captures canonical attestation behaviors,
+that are common to a broad range of attestation-enabled systems. An entity that
+combines multiple Roles produces and consumes the associated Role Messages.
+
+The RATS architecture is not prescriptive about deployment configuration options
+of attestation-enabled systems, therefore the various Roles can be hosted on any
+participating entity. This implies, for a given entity, that multiple Roles could
+be co-resident so that the duties of multiple roles could be performed simultaneously.
+Nevertheless, the semantics of which Role Messages are inputs and outputs to a Role
+entity remains constant. As a result, the entity can participate as a constituent of the RATS architecture
+while flexibly accommodating the needs of various deployment architectures.
+
+Interactions between Roles do not necessarily require use of Internet protocols.
+They could, for example, use inter-process communication, local system buses,
+shared memory, hypervisors, IP-loopback devices or any communication path
```

Comment:
re "IP-loopback devices"... if they use "IP" loopback devices, how do they not use "Internet protocols" as in line 670?

Reply:
True